doc/ref_arch/kubernetes/chapters/chapter04.md

```diff
@@ -5,29 +5,49 @@
 
 ## Table of Contents
 * [4.1 Introduction](#4.1)
-* [4.2 Host OS](#4.2)
+* [4.2 Container Host](#4.2)
```

Comment:
```suggestion
* [4.2 Container Host OS](#4.2)
```

Reply:
To match with naming below

Reply:
The naming below is just `Container Host`?  Which line do you mean?

Reply:
line 23

Reply:
ah thanks - I've corrected that line now - ta